src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/LicenseAcceptanceWindow.xaml

```diff
@@ -99,82 +98,73 @@
       </DataTemplate>
     </ResourceDictionary>
   </Window.Resources>
-  <Grid>
-    <Grid.RowDefinitions>
-      <RowDefinition
+  <ScrollViewer x:Name="sv" CanContentScroll="False" VerticalScrollBarVisibility="Auto" HorizontalScrollBarVisibility="Auto">
+    <Grid x:Name="_licenseAcceptanceGrid" MinWidth="{Binding ElementName=sv, Path=ViewportWidth}" MinHeight="{Binding ElementName=sv, Path=ViewportHeight}"
+          Width="{Binding ElementName=sv, Path=ViewportWidth}" Height="{Binding ElementName=sv, Path=ViewportHeight}">
+      <Grid.RowDefinitions>
+        <RowDefinition
         Height="Auto" />
-      <RowDefinition
+        <RowDefinition />
+        <RowDefinition
         Height="Auto" />
-      <RowDefinition />
-      <RowDefinition
+        <RowDefinition
         Height="Auto" />
-      <RowDefinition
-        Height="Auto" />
-    </Grid.RowDefinitions>
-    <TextBlock
```

Comment:
Removed the header, since it's consuming space with no additional info to the user. Think I should remove the string now, or follow-up?

Reply:
Created https://github.com/NuGet/Home/issues/11643